app/components/slack-context-menu.js

```diff
@@ -108,17 +108,28 @@
 		}));
 	}
 
-	function buildLinkMenu(menu, target) {
+	function buildLinkMenu(menu, target, teamUrl) {
 		if (target.attributes.href && target.attributes.href.value) {
 			var hrefValue = target.attributes.href.value;
-			if (hrefValue.startsWith('http') || hrefValue.startsWith('https')) {
-				menu.append(new gui.MenuItem({
-					label: 'Open Link in Browser',
-					click: function () {
-						gui.Shell.openExternal(hrefValue);
-					}
-				}));
+
+			// Relative URL, let's prepend the team URL.
+			if (hrefValue.startsWith('/')) {
```

Comment:
Upon re-evaluating the test cases for `resolve-link`, it looks like we never tested for `//` either. Maybe it's best to change this `if` to use `hrefValue.startsWith('/') && !hrefValue.startsWith('//')`